components/pipeline-service/development/main-pipeline-service-configuration.yaml

```diff
@@ -1960,6 +1960,136 @@ spec:
                       requests:
                         cpu: 200m
                         memory: 200Mi
+        pipelines-as-code-controller:
+          spec:
+            template:
+              spec:
+                containers:
+                  - name: pac-controller
+                    resources:
+                      limits:
+                        cpu: 500m
+                        memory: 500Mi
+                      requests:
+                        cpu: 500m
+                        memory: 500Mi
+        pipelines-as-code-watcher:
+          spec:
+            template:
+              spec:
+                containers:
+                  - name: pac-watcher
+                    resources:
+                      limits:
+                        cpu: "1"
+                        memory: 4.5Gi
+                      requests:
+                        cpu: "1"
+                        memory: 4.5Gi
+        pipelines-as-code-webhook:
+          spec:
+            template:
+              spec:
+                containers:
+                  - name: pac-webhook
+                    resources:
+                      limits:
+                        cpu: 500m
+                        memory: 400Mi
+                      requests:
+                        cpu: 500m
+                        memory: 400Mi
+        pipelines-console-plugin:
+          spec:
+            template:
+              spec:
+                containers:
+                  - name: pipelines-console-plugin
+                    resources:
+                      limits:
+                        cpu: 100m
+                        memory: 35Mi
```

Comment:
35Mi? Isn't that too low? Even if it is enough, I'll suggest using 100 or 200 MB, which won't really make huge difference on the resources usage.